packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js

```diff
@@ -134,6 +135,153 @@ describe('AppBar component', () => {
     })
   })
 
+  describe('computed showContextActions', () => {
+    it('should be false if isTrashPersonalActive is true', () => {
+      const store = createStore({ selected: [], currentFolder })
+      const route = {
+        ...createLocationTrash('files-trash-personal', {
+          params: {
+            storageId: '1'
+          }
+        }),
+        meta: {}
+      }
+      const wrapper = getShallowWrapper(route, store, { isTrashPersonalActive: true })
+      expect(wrapper.vm.showContextActions).toBeFalsy()
+    })
+    it('should be true if isSpacesProjectLocation is true and item is given', () => {
```

Comment:
I'd recommend nesting `describe(...)` statements with repetitive preconditions (in this case e.g. the `isSpacesProjectLocation is true` part).
So instead of 

```
it('should be true if isSpacesProjectLocation is true and item is given', () => { ... }
it('should be false if isSpacesProjectLocation is true but no item is given', () => { ... }
```

you'd have

```
describe('if isSpacesProjectLocation is true', () => {
    describe('and item is selected', () => {
        it('should be true ', () => { ... }
    }
    describe('and no item is selected', () => {
        it('should be false ', () => { ... }
    }
})
```

It doesn't change the outcome of the test obviously, but makes them easier to read & extend/adapt in my opinion 🤓